src/user-guide/workflow-storage-etc.rst

```diff
@@ -1,25 +0,0 @@
-.. _WorkflowStorageEtc:
-
-Workflow Storage, Discovery, Revision Control, and Deployment
-=============================================================
```

Comment:
Is there not a replacement for this section?

Reply:
I don't think its necessary really. Python (for instance) doesn't bother to document why it doesn't have built in storage and revision control for Python programs - it should be obvious to users.  And it seems to me we probably won't be recommending Rose for the job, at least until some future git-based version, knowing that everyone (even UM Partners) have moved away from svn except for use of the MO SRS.